lib/rules/no-comment-textnodes.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(context) {
 
   return {
     Literal: function(node) {
-      if (/\s*\/(\/|\*)/.test(node.value)) {
+      if (/^\s*\/(\/|\*)/m.test(node.value)) {
```

Comment:
It's already a test https://github.com/yannickcr/eslint-plugin-react/blob/master/tests/lib/rules/no-comment-textnodes.js#L170